api/nakadi-event-bus-api.yaml

```diff
@@ -1084,6 +1090,16 @@ definitions:
       - read_parallelism
       - write_parallelism
 
+  EventTypeOptions:
+    type: object
+    description: |
+      Additional parameters for tuning internal behavior of Nakadi.
+    properties:
+      retention_time:
+        type: long
```

Comment:
Are you sure it's possible to use a type `long`? I thought that we should use `type: integer` and `format: int64` https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md